Gemfile

```diff
@@ -1,9 +0,0 @@
-# This gemfile provides additional gems for testing and releasing this cookbook
-# It is meant to be installed on top of ChefDK / Chef Workstation which provide the majority
-# of the necessary gems for testing this cookbook
-#
-# Run 'chef exec bundle install' to install these dependencies
-
-source 'https://rubygems.org'
-
-gem 'community_cookbook_releaser'
```

Comment:
We don't want the Gemfile anymore? It's useful for development unless that dependency is no longer a dependency.

Reply:
This dependency is not used anymore and any gems the cookbook needs should be put in metadata or the resources themselves.